src/Modules/Windows-Core/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -24,8 +24,8 @@ CmdletsToExport= "Format-List", "Format-Custom", "Format-Table", "Format-Wide",
     "Unblock-File", "Get-Runspace", "Debug-Runspace", "Enable-RunspaceDebug", "Disable-RunspaceDebug",
     "Get-RunspaceDebug", "Wait-Debugger" 
 FunctionsToExport= "Get-FileHash", "New-TemporaryFile", "New-Guid", "Format-Hex", "Import-PowerShellDataFile",
-    "ConvertFrom-SddlString"
-AliasesToExport= "fhx"
+    "ConvertFrom-SddlString", "Get-Uptime"
+AliasesToExport= "fhx", "uptime"

```

Comment:
yep, and you can still invoke it as `uptime` on systems without a native version, due to the last-ditch command search dropping the verb and searching for noun for "get" commands.